spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -115,6 +116,7 @@ public static void configureServerWithArmeriaSettings(
         requireNonNull(metricCollectingServiceConfigurators, "metricCollectingServiceConfigurators");
 
         configurePorts(server, settings.getPorts());
+        configureSettings(server, settings);
```

Comment:
I think that users may confused when their configurator bean is suppressed because of properties. That is, it would be better to make the configurator beans have higher priority than properties.

Reply:
Makes sense.

Reply:
How about adding a test that indicates configurator beans have higher priority?
We also need a test that checks whether the newly added properties are set. We could probably build a `server` and check the `serverConfig`.

Reply:
Thanks. Actually I couldn't find a way to test configuring a `Server` (I didn't think `ServerConfig` at all). And now I know thanks to your information.

Reply:
Oh my god, I missed spring-boot 1.x 😅 Let me fix tests tomorrow.